Doc/library/typing.rst

```diff
@@ -348,7 +377,10 @@ A generic type can have any number of type variables. All varieties of
    B = TypeVar('B', bound=Sequence[bytes], covariant=True)
    S = TypeVar('S', int, str)
 
-   class WeirdTrio(Generic[T, B, S]):
+   class OldWeirdTrio(Generic[T, B, S]):
+       ...
+
+   class NewWeirdTrio[T, B: Sequence[bytes], S: (int, str)]:
        ...
```

Comment:
I'd prefer it if these type variables had different names to the ones for `OldWeirdTrio`, or it might look like this class is reusing the same type variables from the global scope.

Also, should the new version come first, maybe?